app/src/@libs/evm-wallet/constants.ts

```diff
@@ -43,17 +43,17 @@ export const chainConfigurations: { [K in EvmChainId]?: ChainConfiguration } = {
   },
 };
 
-export const supportedChainIds = [
+export const supportedChainIds: EvmChainId[] = [
   // EvmChainId.ETHEREUM,
   // EvmChainId.ETHEREUM_ROPSTEN,
   EvmChainId.AVALANCHE,
   EvmChainId.AVALANCHE_FUJI_TESTNET,
-] as const;
+];
 
-export const supportedChainsRps: Record<
-  typeof supportedChainIds[number],
-  string
-> = {
+export const supportedChainsRps: Record<EvmChainId, string> = {
```

Comment:
should be "Rpcs" not Rps

Reply:
✅ https://github.com/Anchor-Protocol/anchor-web-app/pull/314/commits/3c504715e1d9ab2eb6029d564857e957007c9f08